protocol/common/message.go

```diff
@@ -627,6 +627,18 @@ func (m *Message) PrepareContent(identity string) error {
 	return m.parseAudio()
 }
 
+func (m *Message) IsSystemMessage() bool {
+	return m.ContentType == protobuf.ChatMessage_UNKNOWN_CONTENT_TYPE ||
```

Comment:
Should Unknown really be considered a system message?

Reply:
Good point. Removed it from the list.